presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -818,6 +829,11 @@ public Plan createPlan(Session session, @Language("SQL") String sql, List<PlanOp
     }
 
     public Plan createPlan(Session session, @Language("SQL") String sql, List<PlanOptimizer> optimizers, LogicalPlanner.Stage stage)
```

Comment:
Who uses this variant?

Reply:
com.facebook.presto.sql.planner.assertions.BasePlanTest. 
Will remove this variant.